wmbc/wmbc.py

```diff
@@ -176,6 +193,12 @@ async def main():
             type=str,
             help='File with binary modbus frame needed for modbus_1s or modbus_p commands'
     )
+    parser.add_argument(
```

Comment:
Actually there is no need for that, when running WMBC as main() we should just rely on files, `modbus_frame` should be only used combined with modbus frame generation.